test/claims-status/components/appeals-v2/Alert.unit.spec.jsx

```diff
@@ -3,18 +3,41 @@ import { shallow } from 'enzyme';
 import { expect } from 'chai';
 import Alert from '../../../../src/js/claims-status/components/appeals-v2/Alert';
 
-const defaultProps = {
-  alert: {
-    type: 'hearing-scheduled',
-    details: {
-      date: 'March th, 2018'
-    }
-  }
-};
+const defaultProps = {};
```

Comment:
Does it still make sense to have this here?

Reply:
Perhaps not, but wanted to be explicit about what was going on